test/integration/users.test.js

```diff
@@ -1161,4 +1161,21 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /update-in-discord", function () {
+    it("it returns proper response", function (done) {
```

Comment:
- What does "proper response" mean?

Reply:
It means a not failing, or the mentioned response. 
For a one-time script do we need to focus on the test title as well?